examples/meta/generator/targets/cpp.json

```diff
@@ -90,7 +90,10 @@
         "IntLiteral": "$number",
         "RealLiteral": "$number",
         "FloatLiteral": "${number}f",
-        "MethodCall": "$object->$method($arguments)",
+        "MethodCall": {
+            "Default": "$object->$method($arguments)",
+            "get_real": "$object->get<float64_t>($arguments)"
+        },
         "StaticCall": "C$typeName::$method($arguments)",
         "GlobalCall": "$method($arguments)",
```

Comment:
Do you think you would need something similar for `StaticCall` and `GlobalCall`? Can easily implement haven't done it for now

Reply:
I think this should suffice for now